platform/platform_tests/distance_tests.cpp

```diff
@@ -0,0 +1,132 @@
+#include "testing/testing.hpp"
+
+#include "platform/distance.hpp"
+
+using namespace std;
+
+namespace platform
+{
+UNIT_TEST(Distance_IsValid)
+{
+  TEST_EQUAL(Distance(0.0).IsValid(), false, ());
+  TEST_EQUAL(Distance(-1).IsValid(), false, ());
+  TEST_EQUAL(Distance(numeric_limits<double>::epsilon()).IsValid(), false, ());
+  TEST_EQUAL(Distance(1).IsValid(), true, ());
+  TEST_EQUAL(Distance(1243.2, Distance::Units::Feet).IsValid(), true, ());
+}
+
+UNIT_TEST(Distance_Conversions)
+{
+  // TODO
+}
+
+UNIT_TEST(Distance_GetUnits)
+{
+  TEST_EQUAL(Distance(1234).GetUnits(), Distance::Units::Meters, ());
+  TEST_EQUAL(Distance(1234, Distance::Units::Kilometers).GetUnits(), Distance::Units::Kilometers,
+             ());
```

Comment:
Is it good for you (if you are using formaters) put 120 symbols in a line?
It looks ugly, no?

Reply:
I'm using `.clang-format` in repo 😄

Reply:
It looks ugly.